src/components/PostListRow/PostListRow.scss

```diff
@@ -44,7 +44,7 @@ div .post-row:nth-child(1) {
 
 .type-author {
   display: flex;
-  align-items: end;
+  align-items: flex-end;
```

Comment:
This was a listing thing. `end` should be the same as `flex-end` in this case and `end` is marginally supported...